README.md

```diff
@@ -655,10 +655,12 @@ API | Description | Auth | HTTPS | CORS |
 | [IP Location](https://ip-api.com/docs) | Find location with ip address | No | No | Unknown |
 | [IP Sidekick](https://ipsidekick.com) | Geolocation API that returns extra information about an IP address | `apiKey` | Yes | Unknown |
 | [IP Vigilante](https://www.ipvigilante.com/) | Free IP Geolocation API | No | Yes | Unknown |
+| [ip-api](http://ip-api.com/) | Find location with ip address | No | No | Unknown |
```

Comment:
Remove this, please, already listed here:

```
| [IP Location](https://ip-api.com/docs) | Find location with ip address | No | No | Unknown |
```

If you want to rename the one listed to ip-api, I can accept that ;)

Reply:
Oh, I didn't notice that, I removed the one I added :)